### DISCUSSION TITLE
How to set streaming output in AI chat and code generation?

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1330] [Q&A]
I found that I will waiting for a while to return the results, rather than returning the chat content in real time, is there an option to turn on streaming output？

The following two scenarios require waiting
1、
![image](https://github.com/user-attachments/assets/065390b3-86b6-4287-b5a0-dbb6dc5986d1)
2、
![image](https://github.com/user-attachments/assets/15496a69-2bc5-46e0-a8c4-df97d7777eea)

◆
Isn't the markdown content returned in the chat window converted to html，There are also such symbols "  ``` "
![image](https://github.com/user-attachments/assets/11e107a1-9806-4b90-8b59-578fb5e588b2)

**Comment by pyscripter** [comment]
Streaming is not currently supported.

**Reply by wilikeit** [reply]
Okay, looking forward to it being supported in the future.

**Comment by pyscripter** [comment]
> Isn't the markdown content returned in the chat window converted to html，There are also such symbols " ``` "

The triple quotes are used to indicate code blocks as in Markdown.    They are not converted to html.  Just syntax highlighted as in the code editor.

**Reply by wilikeit** [reply]
That must be my misunderstanding. I thought it returned the Markdown source code.
